### ISSUE TITLE
Issues between local denokv DB and KV Connect

### ISSUE THREAD

**Issue by mcgear** [denoland/denokv#57]

**Comment by igorzi** [comment]
Can you please describe your project structure? Do you have a `listenQueue` handler in one Deno Deploy project? and you're trying to enqueue from another Deno Deploy project?

**Comment by mcgear** [comment]
No, everything is in the same project. The index file i linked to is where it starts, and the handler is here:
https://github.com/fathym-deno/everything-as-code/blob/integration/handlers/eac/commit.handler.ts

The helper methods are here:
https://github.com/fathym-deno/everything-as-code/blob/integration/src/utils/deno-kv/helpers.ts

Additional handling for the checks are here:
https://github.com/fathym-deno/everything-as-code/blob/integration/handlers/eac/commit-check.handler.ts

Let me know if you need anything else.

**Comment by igorzi** [comment]
In Deno Deploy, you don't need to use KV connect to interact with a KV database. You can just do `Deno.openKv()` and it will use the default KV database in your project.

Queues are currently not supported in KV connect protocol.

**Comment by mcgear** [comment]
Is there a timeline for KV connect protocol support?  We deploy our deno apps outside of Deno Deploy.

**Comment by igorzi** [comment]
It's on the roadmap. Do you want to only enqueue and dequeue outside of Deno Deploy? Or just enqueue from outside, but have `listenQueue` in Deno Deploy?

**Comment by mcgear** [comment]
Looking to do both outside at this point

**Comment by mcgear** [comment]
Wanted to check in on this and see what the roadmap looked like for this.

**Comment by Briscoooe** [comment]
Checking if there's any updates on getting queues to work with KV Connect?
